setup.py

```diff
@@ -49,6 +49,7 @@ def read_requirements(file_):
             "bugbug-check = scripts.check:main",
             "bugbug-microannotate-generate = scripts.microannotate_generator:main",
             "bugbug-classify-commit = scripts.commit_classifier:main",
+            "bugbug-classify-bug = scripts.bug_classifier:main",
```

Comment:
I saw that there's another PR to add the scripts to setup.py; I added it here, but I can remove it if this is the wrong PR to do this.

Reply:
No, this is the right PR ;)